standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/hive_metastoreConstants.java

```diff
@@ -97,4 +97,10 @@
 
   public static final java.lang.String EXPECTED_PARAMETER_VALUE = "expected_parameter_value";
 
+  public static final java.lang.String HIVE_SQL_JDBC_USERNAME = "hive.sql.dbcp.username";
```

Comment:
Why do we need to define separate var for every JDBC config? 
Can't we create a list with connection properties we'd like to exclude and pass it to a filter?

Reply:
>  Can't we pass hive.sql regex to a filter?

It is a great question: if we add a regex pattern to filter this out, there is a chance that we can filter out something in the future that is not implemented yet. And the person who implements it, won't even know about it at all. But on the other hand, there are no huge improvements at Hive - I mean, it looks like a complete software - so I see only a little chance that it will happen. 

And also, thank you for this comment. I just now realized it is a thrift generated code. So it is definitely a bad idea to put a new constant here - it will be removed as thrift is re-generated. Let me find a proper place to store those constants.